experiments/post_layout_optimization/post_layout_optimization.cpp

```diff
@@ -76,7 +83,8 @@ int main()  // NOLINT
 
         // check equivalence
         fiction::equivalence_checking_stats eq_stats{};
-        fiction::equivalence_checking<fiction::technology_network, gate_lyt>(network, gate_level_layout, &eq_stats);
+        fiction::equivalence_checking<fiction::technology_network, gate_lyt>(benchmark_network, gate_level_layout,
+                                                                             &eq_stats);
```

Comment:
If you wanted, you could bail the `stats` here, too, and rely on the return type.